src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -2302,6 +2352,24 @@ std::error_code Session::receive_error_message(const ProtocolErrorInfo& info)
     return std::error_code{}; // Success;
 }
 
+std::error_code Session::receive_test_command_response(request_ident_type ident, std::string_view body)
+{
+    logger.info("Received TEST_COMMAND: \"%1\" (request_ident=%2)", body, ident);
```

Comment:
you could follow the usual format: `Received: TEST_COMMAND`. And also add the session_ident

Reply:
Done